app/api/v1/endpoints/projections.py

```
@@ -0,0 +1,202 @@
+"""Projection API: definitions and state (Phase 5)."""
+
+from datetime import datetime
+from typing import Annotated, Any
+
+from fastapi import APIRouter, Depends, HTTPException, Query
+
+from app.api.v1.dependencies import (
```

Flake8 (7.3.0)

[error] 'app.api.v1.dependencies.get_tenant_id' imported but unused

(F401)